data/google-analytics.json

```diff
@@ -15,10 +15,18 @@
       "key": "gtag"
     },
     {
-      "code": "window[{{l}}]=window[{{l}}]||[];window['gtag-'+{{l}}]=function (){window[{{l}}].push(arguments);};window['gtag-'+{{l}}]('js',new Date());window['gtag-'+{{l}}]('config',{{id}})",
+      "code": "window[{{l}}]=window[{{l}}]||[];window['gtag-'+{{l}}]=function (){window[{{l}}].push(arguments);};window['gtag-'+{{l}}]('consent', {{consentType}}, {{consentValues}});window['gtag-'+{{l}}]('js',new Date());window['gtag-'+{{l}}]('config',{{id}})",
       "params": ["id"],
       "optionalParams": {
-        "l": "dataLayer"
+        "l": "dataLayer",
+        "consentType": "default",
+        "consentValues": {
+          "ad_user_data": "denied",
+          "ad_personalization": "denied",
+          "ad_storage": "denied",
+          "analytics_storage": "denied",
+          "wait_for_update": 500
+        }
```

Comment:
Are these all the default settings? We should definitely have the default values here and then Nuxt (or any consumer) can update them as they fit

Reply:
Looks like `denied` is the default consent state per the [docs](https://developers.google.com/tag-platform/security/guides/consent?consentmode=advanced#default-consent) 👍